.taskcluster.yml

```diff
@@ -7,8 +7,19 @@ tasks:
     - $let:
           taskgraph:
               branch: taskgraph
-              revision: 2b32db7d303dd175ed4c4abf5fcfe00c6c593acd
+              revision: c78d65237c6ed064b9cbb3cd063b22e9e052a06a
+          template:
+              # XXX uncomment to merge
+              # repo: https://github.com/mozilla-releng/xpi-template
+              # branch: master
+              repo: https://github.com/escapewindow/xpi-template
+              branch: dep-signing
           trustDomain: xpi
+          # XXX for private repos, uncomment this line, and comment out the next
+          # githubCloneSecret: project/xpi/xpi-github-clone-ssh
+          githubCloneSecret: ""
+          # XXX use "privileged" or "system" to enable signing
+          xpiSigningType: ""
```

Comment:
Possibly. I also want to add the expected extension name(s) in there as a string or list of strings. I've avoided modifying package.json to date, since the xpi-manifest is the source of truth for release builds, and the CI provided here is a best effort attempt at making new source repositories super simple to drop in.

Reply:
https://github.com/mozilla-extensions/xpi-manifest/issues/33